qa/tasks/cephfs/test_fragment.py

```diff
@@ -314,3 +314,78 @@ def _count_fragmented():
             lambda: _count_fragmented() > 0,
             timeout=30
         )
+
+class TestDirfragKillPoints(TestFragmentation, CephFSTestCase):
+    # Two active MDS
+    MDSS_REQUIRED = 2
+
+    init = False
+
+    def _setup_cluster(self):
+        # Set Multi-MDS cluster
+        self.fs.set_max_mds(2)
+
+        self.fs.wait_for_daemons()
+
+        all_daemons = self.fs.get_daemon_names()
```

Comment:
why?

Reply:
Oh! I thought I had cleaned this up. Will remove.